openrouteservice/src/main/resources/app.config.sample

```diff
@@ -68,6 +68,7 @@
             "maximum_distance_alternative_routes": 100000,
             "maximum_alternative_routes": 3,
             "maximum_distance_round_trip_routes": 100000,
+            "maximum_speed": 80,
```

Comment:
Should this configuration parameter be used instead of the hard-coded values above?

Reply:
Yes I changed that. Thanks.